src/MICore/Debugger.cs

```diff
@@ -1061,6 +1062,27 @@ private void OnOutOfBand(string cmd)
                 string status = _miResults.ParseCString(cmd.Substring(8));
                 OnStateChanged("stopped", status);
             }
+            else if (cmd.StartsWith("stopped", StringComparison.Ordinal))
+            {
+                if (PlatformUtilities.IsWindows() &&
+                    this.LaunchOptions is LocalLaunchOptions &&
+                    ((LocalLaunchOptions)this.LaunchOptions).ProcessId != 0 &&
+                    this.MICommandFactory.Mode == MIMode.Gdb &&
+                    !this.IsCygwin
+                    )
+                {
+                    // mingw enters break mode with no status during attach. All of the processing 
+                    // for *stopped expects status or it will assume the event is a module load.
+                    // to avoid regressing that code path, manufacture a fake frame. This will not
+                    // be seen by the user as it is continued during attach.
+                    string fakeMingwAttachFrame = "frame={addr=\"0x0000000000000000\",func=\"fakemodule!fakeMingwAttachFrame\"},thread-id=\"1\",stopped-threads=\"all\"";
+                    OnStateChanged("stopped", fakeMingwAttachFrame);
```

Comment:
I think it might do entrypoint processing on this event. Specifically the line in DebuggedProcess.HandleBreakModeEvent if (String.IsNullOrWhiteSpace(reason) && !_bEntrypointHit).

Note setting _bEntrypointHit could be a source of bugs. Interestingly, I'm actually finding that mingw never sends entrypoint for normal cases either.